### ISSUE TITLE
Is react-native  supported?

### ISSUE THREAD

**Issue by kennetpostigo** [cristianbote/goober#139]
Hi 👋
Wanted to know if react-native was supported?

**Comment by cristianbote** [comment]
Hey @kennetpostigo, thanks for opening this issue! 😃 Uh, I really don't think goober will work with React Native anytime soon. I haven't looked into it, but I've never thought for goober to work in React Native.

But, now that I'm thinking about it, goober does transform all the css into an object, that will get serialised into _real_ css. So, basically if instead of updating a `<style />` tag goober would call `StyleSheet.create({ rule: { ...object } })` that'll make it work for sure. But, nesting and such will get weird and would need a proper handler.

TL;DR; don't think so, but if we can somehow cover it in our threshold, why not?

**Comment by cristianbote** [comment]
If there is someone who wants to pick-up the work for RN feel free to reach out. I'll close this for now.
